src/EnergyPlus/WaterUse.cc

```diff
@@ -1058,6 +1058,7 @@ namespace WaterUse {
         // Calculate desired hot and cold water flow rates
 
         Real64 const EPSILON(1.e-3);
+        Real64 TempDiff;
```

Comment:
You can declare and define all three diffs separately here, that way you don't have to calculate them once for the if statements and again within them.